### DISCUSSION TITLE
Official Power Apps Sample code not found

### DISCUSSION THREAD

**Discussion by JimDaly** [rpothin/PowerPlatform-OpenSource-Hub#4] [General]
Microsoft has a lot of Power Apps samples located here: https://github.com/microsoft/PowerApps-Samples
I don't find them here. Can you include them?

**Comment by rpothin** [comment]
Hi @JimDaly,

Thank you for raising this and for your interest to join this initiative 😊

One of the next steps for me will be to document how to self-add a repository in the Power Platform Open-Source Hub, and you will see it is pretty simple 😉

Just add one of the topics listed here to your repository: https://github.com/rpothin/PowerPlatform-OpenSource-Hub/blob/main/Configuration/GitHubRepositoriesSearchCriteria.json

1. In your repository click on the gear in the "About" section

<img width="163" alt="image" src="https://github.com/rpothin/PowerPlatform-OpenSource-Hub/assets/23240245/5eeb5faf-3ee5-40f7-a856-683bedc5d07f">

2. Add the relevant topics in the dedicated field

<img width="330" alt="image" src="https://github.com/rpothin/PowerPlatform-OpenSource-Hub/assets/23240245/007e62c6-1d90-4209-ada4-3351d4434d64">

3. The next morning an automation in this repository will find your repository and you should soon after that appear on the website.

> In your case, I think `powerapps` (_I recommend this one_) or `power-apps` would be good ideas 😊

I hope this helps.
Have a good weekend.

**Reply by JimDaly** [reply]
Thanks. I've done this.

**Reply by rpothin** [reply]
Hey @JimDaly 👋 

Thank you for doing this 😊

I am happy to see the on-boarding process is working as I expected 😁

![image](https://github.com/rpothin/PowerPlatform-OpenSource-Hub/assets/23240245/735c5d78-fe3c-448c-a41c-f0ead5f132dd)

Have a great day.
Raphael
